### PR TITLE
Update falsepositive.list

### PR DESCRIPTION
**Pull request by reachfirst7054**
## Domain/URL/IP(s) where you have found the Phishing:

imunify360.com

### Screenshot

<details><summary>Click to expand</summary>

</details>

### CODE REVIEW

**Comment by reachfirst7054** [comment]
Remove the domains

File: falsepositive.list [hunk 1]
```diff
@@ -22,4 +22,7 @@ salesforce.gompels.com
 stats.sender.net
 app.aave.com
 aave.com
-ipfs.io
\ No newline at end of file
+ipfs.io
```
Comment (spirillen) [inline falsepositive.list:25]: Conflicts with https://github.com/mitchellkrogza/phishing/pull/213

See also issue https://github.com/mitchellkrogza/phishing/issues/226

**Comment by spirillen** [comment]
> Remove the domains

Answer the questions in the issue template, we do not take orders, we handle requests based on knowledge

**Comment by spirillen** [comment]
Closing, also needs rebasing

**Pull request closed without merging**